x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/action/repositories/ClearCcrRestoreSessionActionTests.java

```diff
@@ -55,10 +55,8 @@ public void testPrivilegeForActions() {
 
     public void testActionNames() {
         final ActionFilters actionFilters = mock(ActionFilters.class);
-        final TransportService transportService = mock(TransportService.class);
+        final TransportService transportService = MockUtils.setupTransportServiceWithThreadpoolExecutor();
```

Comment:
I wonder if we want to keep `MockUtils` generic and put all common mocks set up there or is it better to have it `TransportService` specific? If so it could be renamed to `TransportServiceMocks`?